src/import/sst_service.rs

```diff
@@ -217,15 +217,16 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
             let res = compact_files_in_range(&engine, start, end, output_level);
             match res {
                 Ok(_) => info!(
-                    "compact files in range [{:?}, {:?}) to level {:?} takes {:?}",
-                    start,
-                    end,
-                    output_level,
-                    timer.elapsed()
+                    "compact files in range";
+                    "start" => %::hex::encode_upper(start.unwrap_or_else(|| b"")),
```

Comment:
You can use `log_wrappers::DisplayValue` here, like `start.map(DisplayValue)`.

Reply:
`[u8]` doesn't implement `std::fmt::Display`

Reply:
How about `log_wrappers:Key`?